x-pack/plugins/monitoring/server/lib/beats/get_beats.ts

```diff
@@ -25,47 +46,47 @@ export function handleResponse(response, start, end) {
 
     // add another beat summary
     accum.ids.add(uuid);
-    const earliestStats = get(hit, 'inner_hits.earliest.hits.hits[0]._source.beats_stats');
+    const earliestStats = hit.inner_hits.earliest.hits.hits[0]._source.beats_stats;
```

Comment:
I think this might need some safeguarding as well, something like:
```suggestion
    const earliestStats = hit.inner_hits?.earliest?.hits?.hits[0]?._source?.beats_stats;
```